src/PKSim.Core/Snapshots/CompoundProcessSelection.cs

```diff
@@ -1,14 +1,13 @@
-﻿using System.ComponentModel.DataAnnotations;
-using OSPSuite.Core.Domain;
+﻿using OSPSuite.Core.Domain;
 
 namespace PKSim.Core.Snapshots
 {
    public class CompoundProcessSelection : IWithName
    {
-      [Required]
       public string Name { get; set; }
       public string MoleculeName { get; set; }
       public string MetaboliteName { get; set; }
       public string CompoundName { get; set; }
+      public string SystemicProcessType { get; set; }
```

Comment:
Needed to add this property to be able to know what type of process was being saved. In the case of NO mapping, the name is also empty so I removed the required flag